File: DS using c/sorting-techniques/bubble_sort.c
#include<stdio.h>
int i,j;
int temp;
void display(int a[],int n){
	for( i=0;i<n;i++)
		printf("%d ",a[i]);
	printf("\n\n");
}

void swap(int *x,int *y){
    temp=*x;
	*x=*y;
	*y=temp;
}

void bubble_sort(int a[],int n){
	for( i=0;i<n-1;i++){
		for( j=i+1;j<n;j++){
			if(a[i]>a[j])
			swap(&a[i],&a[j]);
		}
	}
	printf("After sorting elements of array are: ");
	display(a,n);
}

void intialize(int n){
	int a[n];
	for( i=0;i<n;i++){
		printf("Enter element-%d: ",i+1);
		scanf("%d",&a[i]);
	}
	system("cls");
	printf("Before sorting elements of array are: ");
	display(a,n);
	bubble_sort(a,n);
}

int main(){
	int n;
	printf("Enter size of the array: ");
	scanf("%d",&n);
	printf("\n");
	intialize(n);
	return 0;
}

File: DS using python/sorting-techniques/bubble_sort.py
def Bubblesort(a, n):
    for i in range(n - 1):
        for j in range(i + 1, n):
            if a[i] > a[j]:
                temp = a[i]
                a[i] = a[j]
                a[j] = temp


array = [int(x) for x in input("Enter elements seperated by space").split()]
Bubblesort(array, len(array))
print("Array after sorting is", array)
